flow/diff_context.cc

```diff
@@ -128,23 +145,32 @@ Damage DiffContext::ComputeDamage(const SkIRect& accumulated_buffer_damage,
   return res;
 }
 
+SkRect DiffContext::MapRect(const SkRect& rect) {
+  SkRect mapped_rect;
+  if (!clip_tracker_.mapRect(rect, &mapped_rect)) {
+    // TODO(knopp): What to do here? We could possibly force repainting
```

Comment:
Ignore it. (You can trace through the call chain to confirm, but...)

The return value is whether the mapped bounds stayed a rect which is only important for knowing if the transformed bounds now contain parallelogram "slop" (or worse, perspective slop). I should have documented that method better.